src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -539,6 +542,27 @@ public boolean hasStarted() {
             return j.getNode(label);
```

Comment:
Note that this method ^^^ _looks_ like it would have taken care of the bug, but in fact is dead code: https://github.com/jenkinsci/jenkins/pull/9442